packages/web-forms/src/components/controls/RankControl.vue

```diff
@@ -0,0 +1,246 @@
+				</svg>
+				<span>{{ props.question.getValueLabel(value)?.asString }}</span>
+			</div>
+
+			<div class="rank-buttons">
+				<button
+					v-if="values.length > 1"
+					:disabled="disabled || (index === 0)"
+					@click="moveUp(index)"
+					@mousedown="setHighlight(index)"
+				>
+					<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 768 768">
+						<path d="M384 256.5l192 192-45 45-147-147-147 147-45-45z" />
+					</svg>
+				</button>
+
+				<button
+					v-if="values.length > 1"
+					:disabled="disabled || (index === values.length - 1)"
+					@click="moveDown(index)"
+					@mousedown="setHighlight(index)"
+				>
+					<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 768 768">
+						<path d="M531 274.5l45 45-192 192-192-192 45-45 147 147z" />
+					</svg>
+				</button>
+			</div>
+		</div>
+	</VueDraggable>
+
+	<ValidationMessage
+		:message="question.validationState.violation?.message.asString"
+		:show-message="touched || submitPressed"
+	/>
+</template>
+
+<style scoped lang="scss">
+@import 'primeflex/core/_variables.scss';
+
+// Variable definition to root element
+.rank-control {
+	--rankSpacing: 7px;
+	--rankBorder: 1px solid var(--surface-300);
+	--rankBorderRadius: 10px;
+	--rankHighlightBackground: var(--primary-50);
+	--rankHighlightBorder: var(--primary-500);
+}
+
+// Overriding VueDraggable's sortable-chosen class
+.sortable-chosen {
+	opacity: 0.9;
+	background-color: var(--surface-0);
+}
+
+.rank-control {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	align-items: flex-start;
+	gap: var(--rankSpacing);
+}
+
+.rank-option {
+	display: flex;
+	flex-wrap: nowrap;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	padding: 8px;
+	border: var(--rankBorder);
+	border-radius: var(--rankBorderRadius);
+	font-size: 1rem;
+	line-height: 17px;
+	color: var(--surface-900);
+	cursor: move;
+
+	.rank-label {
+		display: flex;
+		align-items: center;
+		gap: var(--rankSpacing);
+	}
+}
+
+.moving,
+.fade-moving {
+	background: var(--rankHighlightBackground);
+	border: 2px solid var(--rankHighlightBorder);
+}
+
+.fade-moving {
+	opacity: 0.5;
+}
+
+.rank-buttons {
+	display: flex;
+	gap: var(--rankSpacing);
+
+	button {
+		border: var(--rankBorder);
+		border-radius: var(--rankBorderRadius);
+		background: var(--surface-0);
+		padding: var(--rankSpacing);
+		line-height: 0;
+	}
+
+	button:hover:not(:disabled) {
+		background: var(--rankHighlightBackground);
+		border: 1px solid var(--rankHighlightBorder);
+	}
+
+	button:disabled {
+		background: var(--surface-100);
+		border: none;
+		svg path {
+			fill: var(--surface-300);
+		}
+	}
+}
+
+.disabled .rank-option,
+.disabled button {
+	cursor: not-allowed;
+}
+
+@media screen and (max-width: #{$sm}) {
+	.rank-buttons {
+		display: none;
+	}
+}
```

Comment:
Not sure about this! The most likely way a user would experience it is on a phone. I think the buttons are especially convenient for a touch interface (where drag/drop can be really awkward, even with finesse like an appropriate delay). I am happy to defer this for now, since it'll probably need more UI/UX design attention. But wanted to note that it feels like a gap to me.

Reply:
The reason @alyblenkin and I removed the buttons on mobile devices was to create more space for text and reduce excessive wrapping, which makes reading harder. Buttons take up too much screen space.